runtime/parachains/src/paras_inherent.rs

```diff
@@ -803,81 +763,68 @@ fn apply_weight_limit<T: Config + inclusion::Config>(
 /// they were actually checked and filtered to allow using it in both
 /// cases, as `filtering` and `checking` stage.
 ///
-/// `EARLY_RETURN` determines the behavior.
-/// `false` assures that all inputs are filtered, and invalid ones are filtered out.
-/// It also skips signature verification.
-/// `true` returns an `Err(_)` on the first check failing.
-pub(crate) fn sanitize_bitfields<T: crate::inclusion::Config, const EARLY_RETURN: bool>(
+/// `CHECK_SIGS` determines if validator signatures are checked. If true, bitfields that have an
```

Comment:
Something that is lingering in the back of my head:
```
const VERIFY_SIGS: bool = true;
const SKIP_SIG_VERIFY: bool = false;
```
so they could be used instead of `true` and `false` in the const generic calls.

Reply:
a414390fe6ed023e1b9741540d446643cfebf03b